doc/development/development.rst

```diff
@@ -201,7 +201,7 @@ Implement a new extractor
 SpikeInterface already supports over 30 file formats, but the acquisition system you use might not be among the
 supported formats list (***ref***). Most of the extractord rely on the `NEO <https://github.com/NeuralEnsemble/python-neo>`_
 package to read information from files.
-Therefore, to implement a new extractor to handle the unsupported format, we recommend make a new `neo.rawio `_ class.
+Therefore, to implement a new extractor to handle the unsupported format, we recommend make a new :code:``neo.rawio ` class.
```

Comment:
```suggestion
Therefore, to implement a new extractor to handle the unsupported format, we recommend make a new :code:`neo.rawio ` class.
```
my editor double ticked automatically. I'll fix that unless you want me to change it to an actual link to Neo. Currently it was set up as a hyperlink without the link, which caused all the following lines to get messed up.

Reply:
Thanks @zm711 

I would do the following: 
```suggestion
Therefore, to implement a new extractor to handle the unsupported format, we recommend make a new :code:`neo.rawio.BaseRawIO` class (see `example <https://github.com/NeuralEnsemble/python-neo/blob/master/neo/rawio/examplerawio.py#L44>`_).
```

What do you think?